test/sql/function/numeric/test_trigo.test

```diff
@@ -450,3 +450,10 @@ NULL
 24
 785
 
+statement error
+select asin(-2);
+----
```

Comment:
Can we perhaps add an expected error message here?

Reply:
aye aye